website/docs/r/wafregional_web_acl_association.html.markdown

```diff
@@ -0,0 +1,94 @@
+---
+layout: "aws"
+page_title: "AWS: aws_wafregional_web_acl_association"
+sidebar_current: "docs-aws-resource-wafregional-web-acl-association"
+description: |-
+  Provides a resource to create an association between a WAF Regional WebACL and Application Load Balancer.
+---
+
+# aws\_wafregional\_web\_acl\_association
```

Comment:
Nitpick: The `\` escape character is no longer necessary. Do you mind removing it?